internal/handlers/sjson/array_test.go

```diff
@@ -58,6 +58,63 @@ var arrayTestCases = []testCase{
 			},
 		},
 		j: `[[],"Qg==",true,1627378542123,{},42.13,42,42,"foo",null]`,
+	}, {
+		// Unlike "array_all" case, this one passes the Fuzz checks successfully.
+		// Most notably, it doesn't have a nested array that is rejected by the data validation.
+		name: "array_without_nested",
+		v: convertArray(must.NotFail(types.NewArray(
+			"",
+			false,
+			must.NotFail(types.NewDocument("key", "value")),
+			-32589.300000101,
+			int32(349),
+			int64(-21424182),
+			types.Null,
+			"a string that is somewhat longer",
+		))),
+		sch: &elem{
+			Type: elemTypeArray,
+			Items: []*elem{
+				stringSchema,
+				boolSchema,
+				{
+					Type: elemTypeObject,
+					Schema: &schema{
+						Properties: map[string]*elem{
+							"key": stringSchema,
+						},
+						Keys: []string{"key"},
+					},
+				},
+				doubleSchema,
+				intSchema,
+				longSchema,
+				nullSchema,
+				stringSchema,
+			},
+		},
+		j: `["",false,{"key":"value"},-32589.300000101,349,-21424182,null,"a string that is somewhat longer"]`,
+	}, {
+		name: "array_all",
```

Comment:
Let's name this something other than `array_all` :hugs: `array_wrong_schema`?

Reply:
That's a copy/paste error right there. :)
I renamed it now.